build/common.ps1

```diff
@@ -271,6 +271,8 @@ Function Install-DotnetCLI {
         & $DotNetExe --info
     }
 
+    # install SDK2 runtime as we encounter problems on running dotnet vstest command when only download SDK3.
+    & $DotNetInstall -Runtime dotnet -Version 2.2.3 -i $CLIRoot -NoPath
```

Comment:
Can we do 2.2.x? 
Is that possible to do with the runtime install.

Reply:
Thanks. Changed to a channel 2.2.